app/src/main/java/com/codesoom/assignment/Constant.java

```diff
@@ -0,0 +1,17 @@
+package com.codesoom.assignment;
+
+/**
+ * Class have constants used in application
+ *
+ * @author Taeheon Woo
+ * @version 1.0
+ *
+ */
+public class Constant {
+    static final int port = 8000;
+    static final int backlog = 0;
```

Comment:
Java에서 `static final` 상수는 대문자로 작성하는 관례가 있습니다. 
```suggestion
    static final int PORT = 8000;
    static final int BACKLOG = 0;
```

자바 공식 튜토리얼 문서의 제일 마지막 부분을 읽어봅시다. 공식 문서를 읽는 것은 개발자에게 훌륭한 훈련이 됩니다.
https://docs.oracle.com/javase/tutorial/java/nutsandbolts/variables.html

> If the name you choose consists of only one word, spell that word in all lowercase letters. If it consists of more than one word, capitalize the first letter of each subsequent word. The names gearRatio and currentGear are prime examples of this convention. If your variable stores a constant value, such as static final int NUM_GEARS = 6, the convention changes slightly, capitalizing every letter and separating subsequent words with the underscore character. By convention, the underscore character is never used elsewhere.